packages/marshal/src/encodePassable.js

```diff
@@ -304,23 +304,19 @@ const decodeTagged = (encoded, decodePassable) => {
  */
 
 /**
- * @param {EncodeOptions} [encodeOptions]
+ * @param {EncodeOptions} encodeOptions
+ * encodeOptions is actually optional, but not marked as such to work around
+ * https://github.com/microsoft/TypeScript/issues/50286
+ *
  * @returns {(passable: Passable) => string}
  */
-// `yarn lint` complains here but not for equivalent code in agoric-sdk.
-// Also, vscode does not complain. Hence we're using at-ts-ignore rather than
-// at-ts-expect-error. Using at-ts-ignore should also generate a complaint
-// that we should be using at-expect-error, where we would normally need
-// to suppress that error as well. However, perhaps that second error currently
-// happens only in agoric-sdk, but not yet in endo. TODO figure out and fix.
-// @ts-ignore
 export const makeEncodePassable = ({
   encodeRemotable = (rem, _) => assert.fail(X`remotable unexpected: ${rem}`),
   encodePromise = (prom, _) => assert.fail(X`promise unexpected: ${prom}`),
   encodeError = (err, _) => assert.fail(X`error unexpected: ${err}`),
 } = {}) => {
   const encodePassable = passable => {
-    if (ErrorHelper.canBeValid(passable, x => x)) {
+    if (ErrorHelper.canBeValid(passable)) {
```

Comment:
Aha. I knew I saw that somewhere but forgot where. Already fixed by https://github.com/endojs/endo/pull/1385 , which is now merged. Sorry for the miscoordination.